test/integration/integration_admin_test.cc

```diff
@@ -55,6 +55,21 @@ TEST_P(IntegrationAdminTest, AdminLogging) {
   EXPECT_EQ("200", request("admin", "POST", "/logging?assert=trace", response));
   EXPECT_EQ(spdlog::level::trace, Logger::Registry::getLog(Logger::Id::assert).level());
 
+  // Multiple loggers at once with bad logger (partial success).
+  EXPECT_EQ("404",
+            request("admin", "POST",
+                    "/logging?paths=blah:debug,assert:debug,admin:debug,config:debug", response));
```

Comment:
can we check what error the user sees?

I think we should return back to the user some text indicating what the problem was (though not echoing back whatever they typed as that can risk XSS).